src/cocotb_tools/makefiles/Makefile.inc

```diff
@@ -72,6 +72,11 @@ define check_for_results_file
     @test -f $(COCOTB_RESULTS_FILE) || (echo "ERROR: $(COCOTB_RESULTS_FILE) was not written by the simulation!" >&2 && exit 1)
 endef
```

Comment:
`check_for_results_file` should not be needed anymore?

Reply:
For compatibility reasons maybe better reuse `check_for_results_file`. In other case should probably keep `check_for_results_file` as is (it will not be covered).

Reply:
Yeah I was going to leave it for backwards compatibility, who knows what users have done. Perhaps I should also add a deprecation warning.

Reply:
I think is ok,